testdata/helm/memcached-operator/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM quay.io/operator-framework/helm-operator:v1.0.0
+FROM quay.io/operator-framework/helm-operator:v1.0.1
```

Comment:
why we are bumping the helm-operator:v1.0.1?

Reply:
Fixed it. My bad, had messed this up.